CRM/Core/Invoke.php

```diff
@@ -348,7 +348,17 @@ public static function form($action, $contact_type, $contact_sub_type) {
    * @param CRM_Core_Smarty $template
    */
   public static function statusCheck($template) {
-    if (CRM_Core_Config::isUpgradeMode() || !CRM_Core_Permission::check('administer CiviCRM')) {
+    $permissions = [];
+    // Transitional arrangement until end of 2019, we have added a new permission
+    // view status checks and if CIVICRM_DISABLE_TRANSITION_STATUS_CHECKS is not defined
+    // or defined as false fall back to standard administer CiviCRM permission
+    if (!CRM_Utils_Constant::value('CIVICRM_DISABLE_TRANSITION_STATUS_CHECKS')) {
+      $permissions[] = ['view status checks', 'administer CiviCRM'];
```

Comment:
@eileenmcnaughton This should handle the transition i would have thought?

Reply:
@seamuslee001 my understanding of @totten's proposal is this - https://github.com/civicrm/civicrm-core/pull/16482 - note that if we go that way the removing this permission (or any others) from Administer CiviCRM is not in the plan

Reply:
@eileenmcnaughton I was under the impression that it was all about removing permissions from administer civicrm so you could then more finer grained allow for access by specific permission be it system or data. I have had a client request this style of functionality and I'm not sure if we are going to resolve #16482 any time soon tbh and this does implement the strategy you had requested in your previous comment of falling back by default to allowing for people with administer CiviCRM to see the status checks as they have before. I can certainly go either way but I guess i feel like we should be doing something more here.

Reply:
Well when I thought through what I understood @totten to be suggesting it was NOT removing permissions from administer CiviCRM - but rather creating new permissions that would allow us not to give Administer CiviCRM to users unless we really wanted them to have 'root' permission